lib/UserInterface/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ endif()
 
 add_cling_library(clingUserInterface
   UserInterface.cpp
+  ClingCodeCompleteConsumer.cpp
```

Comment:
Do we actually still need ClingCodeCompleteConsumer? Or can we just use the default clang::PrintingCCC?